bus-mapping/src/circuit_input_builder/tracer_tests.rs

```diff
@@ -1695,15 +1695,7 @@ fn create_address() {
     // Set up call context at CREATE
     builder.tx_ctx.call_is_success.push(false);
     builder.state_ref().push_call(mock_internal_create());
-    builder.builder.sdb.set_account(
-        &ADDR_B,
-        Account {
-            nonce: Word::from(1),
-            balance: Word::zero(),
-            storage: HashMap::new(),
-            code_hash: Hash::zero(),
-        },
-    );
+    builder.builder.sdb.set_account(&ADDR_B, Account::zero());
```

Comment:
not sure whether this is correct

Reply:
it's no longer correct with your fix. i'm reverting it now.